lib/schema_dev/github_actions.rb

```diff
@@ -0,0 +1,264 @@
+                {
+                  uses: 'actions/checkout@v2',
+                },
+                {
+                  name: 'Set up Ruby',
+                  uses: 'ruby/setup-ruby@v1',
+                  with: {
+                    'ruby-version':  '${{ matrix.ruby }}',
+                    'bundler-cache': true,
+                  },
+                },
+                {
+                  name: 'Run bundle update',
+                  run:  'bundle update',
+                },
+              ],
+      test:   [
+                {
+                  name: 'Run tests',
+                  run:  'bundle exec rake spec',
+                }
+              ],
+      finish: [
+                {
+                  name: 'Coveralls Parallel',
+                  if:   '${{ !env.ACT }}',
+                  uses: 'coverallsapp/github-action@master',
+                  with: {
+                    'github-token': '${{ secrets.GITHUB_TOKEN }}',
+                    'flag-name':    'run-${{ matrix.ruby }}-${{ matrix.activerecord }}-${{ matrix.db }}-${{ matrix.dbversion }}',
+                    parallel:       true,
+                  }
+                }
+              ],
+    }.freeze
+
+    DB_ENV = {
+      postgresql: {
+        POSTGRESQL_DB_HOST: '127.0.0.1',
+        POSTGRESQL_DB_USER: 'schema_plus_test',
+        POSTGRESQL_DB_PASS: 'database',
+      },
+      mysql2:     {
+        MYSQL_DB_HOST: '127.0.0.1',
+        MYSQL_DB_USER: 'root',
+        MYSQL_DB_PASS: 'database',
+      },
+    }.freeze
+
+    DB_STARTUP = {
+      postgresql: [
+                    {
+                      name: 'Start Postgresql',
+                      if:   %q{matrix.db == 'postgresql'},
+                      run:  <<~BASH
+                        docker run --rm --detach \\
+                          -e POSTGRES_USER=$POSTGRESQL_DB_USER \\
+                          -e POSTGRES_PASSWORD=$POSTGRESQL_DB_PASS \\
+                          -p 5432:5432 \\
+                          --health-cmd "pg_isready -q" \\
+                          --health-interval 5s \\
+                          --health-timeout 5s \\
+                          --health-retries 5 \\
+                          --name database postgres:${{ matrix.dbversion }}
+                      BASH
+                    },
+                  ],
+      mysql2:     [
+                    {
+                      name: 'Start Mysql',
+                      if:   %q{matrix.db == 'mysql2'},
+                      run:  <<~BASH
+                        docker run --rm --detach \\
+                          -e MYSQL_ROOT_PASSWORD=$MYSQL_DB_PASS \\
+                          -p 3306:3306 \\
+                          --health-cmd "mysqladmin ping --host=127.0.0.1 --password=$MYSQL_DB_PASS --silent" \\
+                          --health-interval 5s \\
+                          --health-timeout 5s \\
+                          --health-retries 5 \\
+                          --name database mysql:5.6
```

Comment:
Is there a reason why we're not supporting dbversions here? I think there's 5.6, 5.7 and 8.0 tags [available at dockerhub](https://hub.docker.com/_/mysql).